ee/desktop/client/client.go

```diff
@@ -20,6 +22,12 @@ type client struct {
 	base http.Client
 }
 
+// desktopUserStatus is all the device data sent to the desktop user process
```

Comment:
Do you intend this stuff in this PR? (Okay if so)

Reply:
This is a stub for now, I'll replace this with a real data structure in a separate PR that focuses on just desktop stuff.

Reply:
okay, I wasn't sure how much to review it